app/services/bulk_operation/bulk_revert_applications_to_pending.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class BulkOperation::BulkRevertApplicationsToPending
```

Comment:
If we're already renaming this class, lets drop the superfluous `Bulk` prefix - its already under the `BulkOperations` namespace so I don't think we need the prefix to clarify